### PR TITLE
add: cart page

### PR DESCRIPTION
**Pull request by Sudhanshusoni45**
[Link to Preview](https://deploy-preview-5--nike-ecomm.netlify.app/cartpage/cartpage)

image

### CODE REVIEW

**Comment by nshanbhag20001412** [comment]
image
the button text is going outside.

**Review by Husain01 — APPROVED** [review]
Just a small suggestion: hovering over the buttons could add a little shade of the button color. Else looks good to go.

**Comment by Sudhanshusoni45** [comment]
> 

Yes thank you for letting me know, I Have not worked on responsiveness yet.

**Review by nshanbhag20001412 — APPROVED** [review]

File: CartPage/cartPage.css [hunk 1]
```diff
@@ -0,0 +1,67 @@
+.cart-container {
+  display: flex;
+  /* grid-template-columns: 1fr 1fr; */
```
Comment (RahullRawat) [inline CartPage/cartPage.css]: You can remove this unncessary code

File: CartPage/cartPage.css [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  border-radius: 20px;
+  background: var(--orange-color);
+  display: flex;
+}
+
+.horizontal-card-details {
+  display: flex;
+  flex-flow: column;
+  justify-content: space-around;
+  /* border: green solid; */
```
Comment (RahullRawat) [inline CartPage/cartPage.css]: this one also

File: CartPage/cartPage.css (hunk 1 above)
Comment (RakshaPawar108) [inline CartPage/cartPage.css]: Remove unnecessary code if not needed.

File: CartPage/cartPage.css (hunk 2 above)
Comment (RakshaPawar108) [inline CartPage/cartPage.css]: Same here

**Review by RahullRawat — APPROVED** [review]
Just small suggestion, you can give `cursor : pointer` to cart icon

File: CartPage/cartPage.html [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+        <i class="fas fa-shopping-cart fa-lg"></i>
+      </nav>
+    </header>
+    <div class="cart-container">
+      <div class="horizontal-ecomm-card">
+        <div class="horizontal-card-image-bg">
+          <img
+            class="card-image"
+            src="./jordan-zoom-separate-pf-basketball-shoes-kGPg2r-removebg.png"
+            alt=""
```
Comment (RakshaPawar108) [inline CartPage/cartPage.html]: Suggestion to add an alt attribute to the image as this will be helpful when or if image does not load

**Review by RakshaPawar108 — APPROVED** [review]
Other than the suggested changes, looks good to merge Sudhanshu! Great work

File: CartPage/cartPage.html (hunk 3 above)
Comment (Sudhanshusoni45) [reply CartPage/cartPage.html]: Done! Thank you Raksha for the suggestions.

File: CartPage/cartPage.css (hunk 1 above)
Comment (Sudhanshusoni45) [reply CartPage/cartPage.css]: Thank you Rahull for suggestions!

**Pull request merged**